correct type

jrw429 opened the pull request:
Cast our requested time to unsigned long to be compatible with Arduino millis().

Fixes issue #22

jrw429 committed:
correct type

Cast our requested time to unsigned long to be compatible with
Arduino millis().

jrw429 committed:
argh, tilde-file

jrw429 committed:
quiet compiler warning

Use gcc-specific preprocessor trick to silence the unused parameter
warning. Not a general solution, but I'm not sure if this is a yet
to be implemented feature, or if it is just an oops and should be
removed.

jrw429 committed:
better fix for unused variable

Better way to quiet compiler about unused variable.

caternuson commented on Adafruit_AS7341.cpp:
```diff
@@ -68,6 +68,8 @@ bool Adafruit_AS7341::begin(uint8_t i2c_address, TwoWire *wire,
  */
 bool Adafruit_AS7341::_init(int32_t sensor_id) {
 
+  (void)sensor_id;      // may be in the future
```
Please change the comment here to `// to avoid compiler warning` to better flag that it's a hack to get around a compiler warn. This follows other libraries that have also done this.

jrw429 replied:
Good suggestion. I think this better explains the intent.

jrw429 committed:
better comment

Better comment to explain purpose of useless statement. Just there
to placate the compiler, while still retaining the parameter that
may be used in the future.

caternuson commented:
Thanks. LGTM.

This also takes care of #24
